dockertest/images_unittests.py

```diff
@@ -341,6 +341,15 @@ def test_addr_no_user(self):
         self.assertEqual(user, None)
         self.assertEqual(addr, "address:1234")
 
+    def test_empty_tag(self):
+        test = "myname:"   # note the trailing colon with no actual tag name
```

Comment:
I'm torn. Ditching 1.9 support is quite likely the right thing to do. But I'm not seeing much big push for 1.10, and am starting to get a bad feeling about this. The kind where it's easier to keep dual support in for a while than to graft it on three months from now when someone says "we need to support 1.9 for a little longer". BTDT.